WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -170,7 +170,10 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 			var unsignedCoinJoin = await ProceedWithSigningStateAsync(roundId, registeredAliceClients, outputTxOuts, cancellationToken).ConfigureAwait(false);
 
 			var finalRoundState = await RoundStatusUpdater.CreateRoundAwaiter(s => s.Id == roundId && s.Phase == Phase.Ended, cancellationToken).ConfigureAwait(false);
-			Logger.LogDebug($"Round ({roundId}): Round Ended - WasTransactionBroadcast: '{finalRoundState.WasTransactionBroadcast}'.");
+			var wasTxBroadcast = finalRoundState.WasTransactionBroadcast
+				? $"'True', Coinjoin TxId: ({unsignedCoinJoin.GetHash()})"
+				: "'False'";
```

Comment:
Hard-coding the `True` and `False` value is fine by me, but others might have a different opinion about this.

Also, since it will be a `Debug` log only, writing out it's a `Coinjoin` feels redundant for me.

Reply:
It is a bit awkward but these are log messages - it displays the information so I am OK with it.